src/runtime/containerd-shim-v2/create.go

```diff
@@ -259,3 +272,99 @@ func doMount(mounts []*containerd_types.Mount, rootfs string) error {
 	}
 	return nil
 }
+
+func configureNonRootHypervisor(runtimeConfig *oci.RuntimeConfig) error {
```

Comment:
I think there'll be more to discuss/review, but we'll want to make sure we have a unit test added for this function as well (ie, call function, verify what's put onto filesystem...

Reply:
I can add an integration tests as a follow up. Create an [issue](https://github.com/kata-containers/kata-containers/issues/2607) to track it and please assign it to me.